src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -3142,7 +3142,8 @@ public String toString() {
   public enum InstrumentOption {
     NONE, // No coverage instrumentation is performed
     LINE_ONLY, // Collect coverage for every executable statement.
-    BRANCH_ONLY; // Collect coverage for control-flow branches.
+    BRANCH_ONLY, // Collect coverage for control-flow branches.
+    ADVANCED;
```

Comment:
Please add a short comment describing what ADVANCED means

Reply:
Also I feel like `ADVANCED` isn't descriptive enough. Internally we don't use "advance" optimization. While I can't think of a good name that describes *what* this type of coverage does - I can suggest `PRODUCTION` that describes *where* it should be used: it's intended to be used in production builds that run in end users browsers. 

Any other suggestions/ideas?

Reply:
For now I changed it to `PRODUCTION` to make it less ambiguous but will change it later if a better name comes up.